api/v1alpha4/awsmachine_types.go

```diff
@@ -147,6 +147,9 @@ type AWSMachineSpec struct {
 	// +optional
 	// +kubebuilder:validation:Enum:=default;dedicated;host
 	Tenancy string `json:"tenancy,omitempty"`
+
+	// Tags for tracking the additionalTags related to volume
+	Tags map[string]string `json:"tags,omitempty"`
```

Comment:
Don't think we want an additional field here. We have an existing AdditionalTags API that does what we want.